nixos/modules/services/networking/avahi-daemon.nix

```diff
@@ -175,25 +175,25 @@ in
     publish = {
       enable = lib.mkOption {
         type = lib.types.bool;
-        default = false;
+        default = true;
         description = "Whether to allow publishing in general.";
       };
 
       userServices = lib.mkOption {
         type = lib.types.bool;
-        default = false;
+        default = true;
         description = "Whether to publish user services. Will set `addresses=true`.";
       };
 
       addresses = lib.mkOption {
         type = lib.types.bool;
-        default = false;
+        default = true;
         description = "Whether to register mDNS address records for all local IP addresses.";
       };
 
       hinfo = lib.mkOption {
```

Comment:
This is by default no https://github.com/avahi/avahi/blob/1dade81cbf4d3e3f49784e44e543cd77046c79df/man/avahi-daemon.conf.5.xml.in#L252-L259